scripts/BuildAndTest.ps1

```diff
@@ -52,10 +52,16 @@ function Exit-WithFailureMessage($scriptName, $message) {
 }
 
 if (-not $NoBuild) {
+    Write-Information "Reformatting code..."
+    dotnet format $RepoRoot\src\Microsoft.Security.Utilities.sln
```

Comment:
>format [](http://example.com/codeflow?start=11&length=6)

Nick, do you know whether a straight `dotnet format` achieves a superset of what happens with `dotnet format style` sub-command (or `dotnet format analyzers`)?

Reply:
I believe it does, yes.